bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java

```diff
@@ -122,26 +124,14 @@
     performWarmup();
   }
 
-  // TODO: This, or at least some variant of this, is recommended by the gRPC team. A client project
-  // broke with this option, so we need to debug that issue before enabling this approach.
-//  private static SslContextBuilder createGrpcSslBuilder() {
-//    SslContextBuilder sslBuilder = GrpcSslContexts.forClient();
-//    if (System.getProperty("java.version").startsWith("1.7.")) {
-//      // The grpc cyphers only work in JDK 1.8+.  Use the default system cyphers for JDK 1.7.
-//      sslBuilder.ciphers(null);
-//      LOG.info("Java 7 detected.  Consider Using JDK 1.8+ which has more secure SSL cyphers. "
-//          + "If you upgrade, you'll have to change your version of ALPN as per "
-//          + "http://www.eclipse.org/jetty/documentation/current/alpn-chapter.html#alpn-versions");
-//    }
-//    return sslBuilder;
-//  }
-
-  // The deprecation is caused by SslContext.newClientContext(). We won't need the @SuppressWarnings
-  // once we get GrpcSslContexts to work properly.
-  @SuppressWarnings("deprecation")
+  private static SslContextBuilder createGrpcSslBuilder() {
+    SslContextBuilder sslBuilder = GrpcSslContexts.forClient();
+    sslBuilder.ciphers(null);
```

Comment:
@nmittler suggested that it should always be set to null after an issue was found with the Dataflow environment.